src/__tests__/App.test.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
```

Comment:
- [ OPTIONAL] Kindly remove the eslint disable line and instead fix the issue in the code.